tsconfig.json

```diff
@@ -16,6 +16,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "types": ["node", "vite/client", "vitest/globals"],
+    "baseUrl": ".",
```

Comment:
is this use only for the showcase or for II as well?
if only showcase, then it would be cleaner to have add it in another  `tsconfig` (that inherits this `tsconfig`)

Reply:
yep good point! tbh I wasn't sure how the `compilerOptions` would merge but I'm guessing it doesn't do anything stupid